test/integration/testenv/docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       volumes:
         - ../dbinit.sql:/docker-entrypoint-initdb.d/init.sql
       ports:
-        - '5432:5432'
+        - '5433:5432'
```

Comment:
Technically this does not need to be exposed at all. Same for some of the other exposed service ports I think. I left them exposed because I found some of them useful when developing the tests. But perhaps they should not be exposed by default?